### PR TITLE
Feature add grid

### PR DESCRIPTION
**Pull request by GuenoleK**
### Description

Adding a grid exemple with the grid and the column components

### CODE REVIEW

File: src/common/grid/example/index.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const Grid = FocusComponents.common.grid;
+const Column = FocusComponents.common.column;
+
+const GreySquare = React.createClass({
+    render() {
+        const label = this.props.label;
```
Comment (pierr) [inline src/common/grid/example/index.jsx]: @GuenoleK  You could have done a pure function here `const GreySquare = ({label}) => <div style={{backgroundColor: '#BDBDBD', height: '200px', color: 'white'}}>{label}</div>`

Comment (pierr) [inline src/common/grid/example/index.jsx]: This is not so important.

**Comment by GuenoleK** [comment]
Thanks for the tip @pierbrisson !

File: src/common/grid/example/index.jsx (hunk 1 above)
Comment (GuenoleK) [inline src/common/grid/example/index.jsx]: Thakns for the tip @pierr !

**Pull request merged**